pkg/controller/capacitycontroller/capacitycontroller.go

```diff
@@ -151,6 +151,8 @@ func (d *Controller) createOrUpdateCapacity(ctx context.Context, drive api.Drive
 		// If ac is exists, update its size to drive size
 		if ac.Spec.Size != size {
 			ac.Spec.Size = size
+			// if size does not match, then probably AC was used for lvg
+			ac.Spec.StorageClass = util.ConvertDriveTypeToStorageClass(drive.GetType())
```

Comment:
Isn't better to check that `ac.Spec.StorageClass` not equals to `util.ConvertDriveTypeToStorageClass(drive.GetType())` and update if not?

Reply:
check added